demo/src/app/components/datepicker/demos/i18n/datepicker-i18n.ts

```diff
@@ -27,6 +27,11 @@ export class CustomDatepickerI18n extends NgbDatepickerI18n {
   getWeekdayShortName(weekday: number): string {
     return I18N_VALUES[this._i18n.language].weekdays[weekday - 1];
   }
+
+  getWeekdayName(weekday: number): string {
+    return I18N_VALUES[this._i18n.language].weekdays[weekday - 1];
+  }
+
```

Comment:
I would have actually call the new method from all the old ones instead of just duplicating.

```typescript
getWeekdayShortName(weekday: number): string {
    return this.getWeekdayName(weekday);
}
```